globalbibletools/platform
# feat: profile view

**What's Changed:**
- Added the `[locale]/profile` path

**QA:**
1. Login. 
2. Go to the profile page. Confirm that the page autofills with the users information.
3. Type in a password of less than 8 characters. Confirm you get a form error
4. Type in a password of 8 characters. Type in a different password into the confirm password input. Confirm you get a form error
5. Type the same password in both boxes. Confirm the form submits properly
